### ISSUE TITLE
Unable to scroll on touch screens

### ISSUE THREAD

**Issue by mashirozx** [Chevereto/Chevereto-Free#31]
Noticed there is a new drag to select feature in picture gallery, but seems this lead to scroll on mobile phone touch screen invalid.

I mean this~~~
![image](https://user-images.githubusercontent.com/16148054/39010296-c6e59dec-4440-11e8-85d5-3329c9d11f07.png)

**Comment by mashirozx** [comment]
Nope, seems your site work fine... You may check the problem [here](https://view.moezx.cc/explore/recent).
I know as a free user I shouldn't ask for anything, but what I met really seems like a bug... Cause I just got this problem after updating it to the latest version hours ago...
Anyway chevereto is absolutely the best image hosting program, thank you~~ :smiley:

**Comment by mashirozx** [comment]
Oh, I just find that the invalid just happen when log in with an admin account ;-)

**Comment by rodolfoberrios** [comment]
Thing is that Chevereto Free is always 6 months behind the paid edition, that's why I asked if the demo has this issue. If don't, then is just a matter of time to get that one fixed.

I know that is quite annoying, but that's the price of the free edition.

In any case, Chevereto v4 will be completely Open Source (no more paid edition) and I'm already working on that.

**Comment by mashirozx** [comment]
Fine, hank you!

**Comment by mashirozx** [comment]
Uhh, I didn't realize this is a individual work before... Great job!

**Comment by bykidi** [comment]
@rodolfoberrios this is crazy... nicely crazy
what made you to move to opensource?

**Comment by rodolfoberrios** [comment]
@bykidi 

Mainly because Chevereto has grown beyond my capacity and I can't fulfill all the new feature request that it gets. Rather than code stuff that I don't like I choose to open my software it so anyone can help to fill the gaps.

Chevereto was founded under the Open Source umbrella and I changed it to paid software to get development ongoing (I got few bucks out of donations) but now is different, Chevereto has a decent install base that makes possible the Open Source model for me. Next July I predict that Chevereto Free installs will match the number of paid + pirated installs.

It is also a shame that few people know about Chevereto just because is paid sofware and that scares people. The main goal of Chevereto is to spawn countless image hosting services, suitable for any kind of media related task and that can be only accomplished if I change it to Open Source and get a bigger install base.

Basically, I think that Chevereto has the potential to seriously re-shape how we deal with image hosting.
